driver/CMakeLists.txt

```diff
@@ -109,8 +109,9 @@ set(DRIVER_SOURCES
 	ppm_ringbuffer.h
 	ppm_syscall.h
 	syscall_table.c
+	syscall_table32.c
```

Comment:
it seems like if `CONFIG_IA32_EMULATION` is defined we include both `<asm/unistd_32.h>` and  `<asm/unistd.h>`, are we sure this won't cause conflicts?

Reply:
We never include both in the same file. In fact, this is the reason we have separate 32 and 64 bit syscall table files.